### ISSUE TITLE
ValueError: unitcell angle < 0

### ISSUE THREAD

**Issue by d-beltran** [mdtraj/mdtraj#1730]
Hi everyone,

I'm getting this error (ValueError: unitcell angle < 0) when saving trajectories.
It happens with different trajectory formats and it is quite random since sometimes it happens and sometimes not. All I have to do in order to avoid the error is retry saving the trajectory 2-3 times until it gets saved.

I am using mdtraj 1.9.7 inside a conda environment.

Thanks for maintaining mdtraj :)

**Comment by apayne97** [comment]
Hi @d-beltran , so sorry for the delay on this! The responsibility of maintaining MDTraj has been transitioning (more info to come soon!). Did you resolve this issue or is this still a problem?

**Comment by d-beltran** [comment]
Hi @apayne97,

What I am doing to avoid the problem now is the following:
```python
trajectory.unitcell_lengths = [0,0,0]
trajectory.unitcell_angles = [90,90,90]
```

It works most times but I am skeptical this is a real fix.
Since the error was happening randomly it is not easy to reproduce so it may still there and it is not happening by luck.

Regards

**Comment by apayne97** [comment]
Right, that looks like somewhat a logical fix (enforcing the angles). 
What are you using to run your simulations? I don't understand how you could have `unitcell_lengths` of 0

**Comment by mattwthompson** [comment]
I'm skeptical that trajectories would fail to save a few times but succeed after retries of the same file. Am I misunderstanding something? Could you post some files that reproduce this behavior? Otherwise we're just guessing.

**Comment by d-beltran** [comment]
There is no misunderstanding. That's how it was.
Right now I do not have a specific file to prove it but I keep all my trajectories. Give me a couple of days. I'll start loading and saving all trajectories until I found the buggy ones and I'll post them here.

**Comment by mattwthompson** [comment]
No apology necessary - if you seem to run into this in the future let us know
